cli.md

```diff
@@ -1,5 +1,7 @@
 # Command Line Interface (CLI)
 
+:information_source: If a parameter supports multiple values they have to be specified via a comma separated list.
```

Comment:
Command line options cannot be repeated. To specify multiple values, provide a single string with comma-separated items (e.g. `-I group1,group2,group3` instead of `-I group1 -I group2 -I group3`).

Reply:
Better not use emoji short names, they won't be rendered in the final output. I use `**NOTE:**` for sidebars which isn't as pretty but clear.